pandas/core/arrays/base.py

```diff
@@ -407,6 +409,7 @@ def isna(self) -> ArrayLike:
 
         * ``na_values._is_boolean`` should be True
         * `na_values` should implement :func:`ExtensionArray._reduce`
+        * `na_values` should implement :func:`ExtensionArray._accumulate`
```

Comment:
Is this necessarily true?

The previous comment is there because we do something like `Series.array.isna()._reduce('any')` (not sure where).